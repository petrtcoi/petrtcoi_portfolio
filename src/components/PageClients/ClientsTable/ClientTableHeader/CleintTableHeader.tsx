import React from 'react'
import { Theme } from '../../../../theme/theme.enum'

export type ClientTableHeaderProps =  {}

const ClientTableHeader: React.FC<ClientTableHeaderProps> = (_props) => {

  return (
    <thead data-testid="table-clients__header">
      <tr>
        <th className="td__left">Full Name</th>
        <th>Gender</th>
        <th className="hide-on-small">Email</th>
        <th>Country</th>
        <th>FlightsQnty</th>
      </tr>
    </thead>
  )
}

export default ClientTableHeader