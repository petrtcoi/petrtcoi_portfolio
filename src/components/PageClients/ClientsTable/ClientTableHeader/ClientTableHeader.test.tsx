import { render, screen } from '@testing-library/react'

import * as story from './CleintTableHeader.stories'

test('should render TableHeader в DarkMode Desktop', () => {
  render(<story.DesktopDark />)
  expect(screen.getByTestId('table-clients__header')).toBeVisible()
})

test('Snapshot TableHeader в DarkMode Desktop', () => {
  expect(screen.getByRole('table')).toMatchSnapshot()
})