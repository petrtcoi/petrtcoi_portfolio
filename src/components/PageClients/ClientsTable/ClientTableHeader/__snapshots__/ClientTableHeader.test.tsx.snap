// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot TableHeader Ð² DarkMode Desktop 1`] = `
<table>
  <thead
    data-testid="table-clients__header"
  >
    <tr>
      <th
        class="td__left"
      >
        Full Name
      </th>
      <th>
        Gender
      </th>
      <th
        class="hide-on-small"
      >
        Email
      </th>
      <th>
        Country
      </th>
      <th>
        FlightsQnty
      </th>
    </tr>
  </thead>
</table>
`;
